Parent/Parent/Conversations/ConversationDetailViewController.swift

```diff
@@ -0,0 +1,105 @@
+//
+// This file is part of Canvas.
+// Copyright (C) 2019-present  Instructure, Inc.
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as
+// published by the Free Software Foundation, either version 3 of the
+// License, or (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+//
+
+import Foundation
+import Core
+
+class ConversationDetailViewController: UIViewController {
+
+    @IBOutlet weak var tableView: UITableView!
+    var conversationID: String!
+    let env = AppEnvironment.shared
+    let refreshControl = UIRefreshControl()
+    var userMap = [String: ConversationParticipant]()
+    var observerID: String = ""
```

Comment:
I think this is also not a parent-specific thing, maybe the `localizedAudience` could just look up the AppEnv current user id? or the update function could look it up, since it doesn't seem to be used elsewhere.